bgpd/bgp_routemap.c

```diff
@@ -6056,6 +6077,46 @@ DEFUN_YANG (no_set_aspath_exclude,
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(set_aspath_exclude_access_list, set_aspath_exclude_access_list_cmd,
+	   "set as-path exclude as-path-access-list WORD$aspaclname",
```

Comment:
AS_PATH_FILTER_NAME please (autocompletion to work)

Reply:
taken into account